ImageSaver/app/src/main/java/com/example/jong/imagesaver/Image_query.java

```diff
@@ -0,0 +1,195 @@
+package com.example.jong.imagesaver;
+
+import android.app.Activity;
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.net.Uri;
+import android.os.Bundle;
+import android.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageButton;
+
+
+import com.example.jong.imagesaver.db.Dbsetup;
+import com.example.jong.imagesaver.db.FeedReaderDbHelper;
+
+import java.util.ArrayList;
+
+
+public class Image_query extends Fragment {
+    private WebView webView;
+    private EditText editText;
+    public static ArrayList<String> listurl = new ArrayList<>();
+    private int pointer;
+    private OnFragmentInteractionListener mListener;
+    private int dbposition=1;
+    FeedReaderDbHelper mDbHelper = MainActivity.mDbHelper;
+    SQLiteDatabase writabledb = mDbHelper.getWritableDatabase();
+
+    //Setting up for writable database
+
+       public static Image_query newInstance(String param1, String param2) {
+        Image_query fragment = new Image_query();
+        Bundle args = new Bundle();
+         return fragment;
+    }
+
+    public Image_query() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+     }
+
+
+
+    public void create_button(View v, String button, final FeedReaderDbHelper mDbHelper, final SQLiteDatabase writabledb){
+        switch(button){
```

Comment:
It seems like you would want to call all of these in your onCreateView so that all of the buttons on screen are active in which case the switch statement is irrelevant.